recipes/thestage-flux/form.json

```diff
@@ -0,0 +1,63 @@
+{
+  "title": "TheStage AI Elastic Models: Flux 1 Schnell",
+  "description": "$replace",
+  "type": "object",
+  "required": [
+    "container_group_name",
+    "huggingface_access_token",
+    "thestage_auth_token",
+    "auth_token"
+  ],
+  "properties": {
+    "container_group_name": {
+      "title": "Container Group Name",
+      "description": "Must be 2-63 lowercase letters, numbers, or hyphens. Cannot start with a number or start or end with a hyphen.",
+      "type": "string",
+      "maxLength": 63,
+      "minLength": 2,
+      "pattern": "^[a-z][a-z0-9-]{0,61}[a-z0-9]$",
+      "default": ""
+    },
+    "autostart_policy": {
+      "title": "Autostart Policy",
+      "description": "Whether to automatically start the container group on deployment.",
+      "type": "boolean",
+      "default": true
+    },
+    "replicas": {
+      "title": "Replicas",
+      "description": "Number of container replicas to run.",
+      "type": "number",
+      "minimum": 1,
+      "maximum": 100,
+      "default": 1
+    },
```

Comment:
```suggestion
```
We automatically prompt for these in the portal, so we don't actually need them in the form.